opencti-platform/opencti-front/src/private/components/analyses/external_references/ExternalReferenceFileImportViewer.tsx

```diff
@@ -103,16 +106,23 @@ ExternalReferenceFileImportViewerBaseProps
   const importConnsPerFormat = scopesConn(connectorsImport);
   const handleOpenImport = (file: FileLine_file$data | null | undefined) => setFileToImport(file);
   const handleCloseImport = () => setFileToImport(null);
-  const onSubmitImport: FormikConfig<{ connector_id: string, configuration: string }>['onSubmit'] = (
+  const onSubmitImport: FormikConfig<{ connector_id: string, configuration: string, objectMarking: Option[] }>['onSubmit'] = (
```

Comment:
configuration & objectMarkings are not always defined (for other connectors than CSV) :
![image](https://github.com/OpenCTI-Platform/opencti/assets/668192/2b4fbaf9-47e9-4f38-8ce8-1c933336b17d)

![image](https://github.com/OpenCTI-Platform/opencti/assets/668192/3be93f47-b394-4a01-adfd-4fa5f333040f)

Reply:
Fix on every launch an import dialogs